docs-v2/customize/guides/advanced/validate-input-and-output.mdx

```diff
@@ -0,0 +1,157 @@
+---
+title: 'Validate input and output'
+sidebarTitle: 'Validate input and output'
+description: 'How to automatically validate your input and ouput with JSON schema and custom alternatives'
+---
+
+Nango provides a straightforward way to validate your data using automatically generated schema files. This guide will walk you through the essentials of data validation in Nango.
+
+## Auto-Generated schema files
```

Comment:
I would start with the simplest content, and describe internals later. 

So I would start with your Automatic validation section, along the lines of "this is so simple, everything is automagic". Then describe how it works with auto-generated schema files. Otherwise it will look scary IMO.